CACI-MIlMOVE/mymove
# B 16771 okta login

## [Agility Ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a808920)

## Summary

As we have had some issues connecting to chamber, we have temporarily stored Okta keys in local environment variables. This PR is not a finalized solution to Okta logins; however, it's to set a new state for development to branch off of. This branch and PR will allow for successful Okta OIDC login and redirect back to the application (All 3 subdomains). Tests are still in progress, and many do not currently work. Again, this is just to set a new state for development branches.

## Verification Steps for Reviewers

Successful Okta login redirects on Eula accept.

### How to test

1. Run the client and server
2. On landing page, accept the Eula to be prompted for Okta login
3. You should be redirected back to Milmove
